doctypes/rng/base/mapGroupDomain.rng

```diff
@@ -552,6 +555,102 @@ PUBLIC "-//OASIS//ELEMENTS DITA 2.0 Map Group Domain//EN"
       </define>
     </div>
   </div>
+      <div>
+      <a:documentation>Map resource</a:documentation>
+      <define name="mapresources.content">
+        <optional>
+          <ref name="topicmeta"/>
+        </optional>
+        <zeroOrMore>
+          <choice>
+            <ref name="data.elements.incl"/>
+            <ref name="topicref"/>
+          </choice>
+        </zeroOrMore>
+      </define>
+      <define name="mapresources.attributes">
+        <optional>
+          <attribute name="href"/>
+        </optional>
+        <optional>
+          <attribute name="keyref"/>
+        </optional>
+        <optional>
+          <attribute name="keys"/>
+        </optional>
+        <optional>
+          <attribute name="keyscope" dita:since="1.3"/>
```

Comment:
Seems odd to have dita:since="1.3" on this and on cascade, since this element doesn't exist in 1.3? I think it's better not to have the attribute at all here (I'd already wondered if we should clean those all out for 2.0 elsewhere.)